<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slate theremin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QGroupBox" name="gp_sound_waveform">
         <property name="title">
          <string>Sound waveform</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="waveform_sine">
            <property name="text">
             <string>Sine</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="waveform_distord_sine">
            <property name="text">
             <string>Distorted sine</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="waveform_square">
            <property name="text">
             <string>Square</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="gp_freq_max">
         <property name="title">
          <string>End frequency</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="0">
           <widget class="QSlider" name="slider_freq_max">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>4000</number>
            </property>
            <property name="value">
             <number>2000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_freq_max">
            <property name="text">
             <string>2000 Hz</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="gp_freq_min">
         <property name="title">
          <string>Start frequency</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="0">
           <widget class="QSlider" name="slider_freq_min">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>4000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_freq_min">
            <property name="text">
             <string>100 Hz</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QGroupBox" name="gp_sound_max">
         <property name="title">
          <string>Max sound</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_sound_max">
            <property name="text">
             <string>100 %</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="slider_sound_max">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="gp_feeling">
         <property name="title">
          <string>Feeling</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Sound frequency</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <widget class="QRadioButton" name="sound_update_pen_on_sight">
               <property name="text">
                <string>Update frequency when pen is on sight</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="sound_update_only_pen_touch">
               <property name="text">
                <string>Update frequency when there is contact</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Sound maintain</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QRadioButton" name="sound_when_no_pen">
               <property name="text">
                <string>Maintain sound if no pen is detected</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="sound_when_pen_on_sight">
               <property name="text">
                <string>Maintain sound when pen is on sight</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="sound_no_maintain">
               <property name="text">
                <string>Don't maintain sound</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QGroupBox" name="gp_sound_min">
         <property name="title">
          <string>Min sound</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_sound_min">
            <property name="text">
             <string>0 %</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="slider_sound_min">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="gp_sample">
         <property name="title">
          <string>Sample</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="3">
           <widget class="QPushButton" name="pb_play_stop">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="list_sample"/>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pb_refresh_sample_list">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/open</normaloff>:/icons/open</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="box_playback_speed">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="suffix">
             <string>x</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>3.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>399</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
